### PR TITLE
feat(api): implement 'ipfs cat'

### PR DESCRIPTION
**Pull request by folex**

### CODE REVIEW

File: aqua/ipfs-api.aqua [hunk 1]
```diff
@@ -49,14 +49,17 @@ func get_and_cache(
...
+        <- Ipfs.put(path)
 
 -- Download file `cid` from IPFS node `from` and save it to `node`
 func get_from(node: PeerId, cid: CID, from: Multiaddr) -> IpfsGetResult:
     on node:
-        result <- Ipfs.get_from(cid, from)
-    <- result
+        <- Ipfs.get_from(cid, from)
+
+-- Download file `cid` from IPFS node `from` and save it to `node`
```
Comment (justprosh) [inline aqua/ipfs-api.aqua:59]: Doc comment is the same as for get_from, it's a little confusing

Comment (justprosh) [reply aqua/ipfs-api.aqua:59]: You should definitely mention that it also returns file contents

File: builtin-package/ipfs_effector_config.json [hunk 2]
```diff
@@ -2,7 +2,7 @@
   "name": "ipfs_effector",
   "mounted_binaries":
   {
-    "ipfs": "/usr/bin/ipfs"
+    "ipfs": "/opt/homebrew/bin/ipfs"
```
Comment (justprosh) [inline builtin-package/ipfs_effector_config.json:5]: I think this should be reverted

**Review by justprosh — APPROVED** [review]
lgtm, but maybe add at least one test?

File: aqua/ipfs-api.aqua (hunk 1 above)
Comment (folex) [reply aqua/ipfs-api.aqua:59]: leftover from copy-pasting. thx!

**Pull request merged**